pkg/workloads/apply.go

```diff
@@ -331,6 +332,9 @@ func applyResources(resources []runtime.Object, ctx context.Context, k8sClient c
 		err = k8sClient.Get(ctx, key, existing)
 
 		if err == nil {
+			if reflect.TypeOf(resource_).String() == "*v1beta1.LocalQueue" {
+				continue
+			}
 			logrus.Warnf("%T: %s/%s already exists. Skipping submit. Use --version flag if you really want to create another resource of this kind", resource_, objMeta.GetNamespace(), objMeta.GetName())
 			continue
 		}
```

Comment:
This could be done more idiomatically

```suggestion
		if err == nil {
			if _, ok := resource_.(*kueuev1beta1.LocalQueue); ok {
                       		continue
                        }
			logrus.Warnf("%T: %s/%s already exists. Skipping submit. Use --version flag if you really want to create another resource of this kind", resource_, objMeta.GetNamespace(), objMeta.GetName())
			continue
		}
```

Reply:
Yes, but then you have to import `v1beta1.LocalQueue`

Reply:
True, but it's more _idiomatic_ ;) Let's go with this